happi/backends/qs_db.py

```diff
@@ -12,6 +12,31 @@
 logger = logging.getLogger(__name__)
 
 
+# Declare our motor types
+motor_types = {'MMS': 'pcdsdevices.epics_motor.IMS',
+               'MMN': 'pcdsdevices.epics_motor.Newport',
+               'MZM': 'pcdsdevices.epics_motor.PMC100'}
+
+
+def guess_motor_class(prefix):
+    """
+    Guess the corresponding pcdsdevices.epics_motor class based on prefix
+
+    Parameters
+    ----------
+    prefix : str
+
+    Returns
+    -------
+    device_class : str
+        Type of EpicsMotor. If not, we assume can use pcdsd
+    """
+    for _typ in motor_types:
+        if _typ in prefix:
```

Comment:
Might want to be more robust than this, just in case a motor comes through with a wonky name like `XPP:USR:MZM:AMMN`... contrived example, I know. I suggest checking `prefix.split(':')` and trying to match a piece of it.

Reply:
Actually, forget about this, we have to strip this out when the questionnaire gets updated anyway.